daphne/tests/test_http_response.py

```diff
@@ -72,7 +72,6 @@ def test_headers(self, headers):
         headers=http_strategies.headers(),
         body=http_strategies.http_body(),
     )
-    @settings(perform_health_check=False)
```

Comment:
N.B. f3b5d854ca309dac07436ab0cea7bc3a12c5061b added it and the commit message implies it was just a slow test